src/pages/Home/subcomponents/FriendReadonlyCountry.js

```diff
@@ -163,20 +168,20 @@ const FriendReadonlyCountry = props => {
         fill: "#6E7377",
         stroke: "rgb(100, 100, 100)",
         strokeWidth: 0.75,
-        outline: "none"
+        outline: "none",
       },
       hover: {
         fill: "rgb(180, 180, 180)",
         stroke: "rgb(180, 180, 180)",
         strokeWidth: 0.75,
-        outline: "none"
+        outline: "none",
       },
       pressed: {
         fill: "#a7e1ff",
         stroke: "#a7e1ff",
         strokeWidth: 0.75,
-        outline: "none"
-      }
+        outline: "none",
+      },
     };
 
     if (isCountryIncluded) {
```

Comment:
@GreenWithMV the switch/if block below is likely another area that would benefit from a refactor- but I would also say that's not within scope of this PR 👍